src/main/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationService.java

```diff
@@ -35,5 +35,7 @@ OrganisationResponse updateOrganisation(OrganisationCreationRequest organisation
     void addPbaAccountToOrganisation(Set<String> paymentAccounts, Organisation organisation, boolean pbasValidated,
                                      boolean isEditPba);
 
+    void deletePaymentsOfOrganization(Set<String> paymentAccounts, Organisation organisation);
```

Comment:
_Non-important suggestion_: can we update the name of this method `deletePaymentsOfOrganization` to use the English spelling of the word `Organisation` to keep it in line with the other method names?